addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp

```diff
@@ -0,0 +1,581 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
```

Comment:
Remove _v8 from file name

Reply:
I think I was able to change the name :D